packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -55,7 +53,8 @@ class _ObjectInspectorViewState extends State<_ObjectInspectorView> {
       initialFractions: const [0.20, 0.80],
       children: [
         ProgramExplorer(
-          controller: programExplorerController,
+          controller: controller.programExplorerController,
+          onSelected: _onLocationSelected,
```

Comment:
can you rename the `onSelected` parameter to `onLocationSelected` to make it more clear what the difference is between `onSelected` and `onNodeSelected`

Reply:
on second thought, can we just make the logic in `_onLocationSelected` part of the `_onNodeSelected` function? Let's get rid of the `onSelected` parameter in ProgramExplorer in general and make the existing code that passes `onSelected` use `onNodeSelected` instead.

Reply:
Yes, I think that makes sense.